### ISSUE TITLE
Bech32 compatibility - a must 

### ISSUE THREAD

**Issue by FrancisPouliot** [bitcoin-dot-org/bitcoin.org#2892]
Realistically bech32 compatibility is a must-know criteria when choosing a wallet. Backwards incompatible wallets should start to get phased out from recommendations eventually it they don't update as probably I maintained or below standards software.

**Comment by crwatkins** [comment]
I agree.  I have proposed adding that in stage 3 [here](https://github.com/bitcoin-dot-org/bitcoin.org/issues/2861#issue-411244142).

**Comment by luke-jr** [comment]
I'm not sure "receive" makes sense as a category. What matters is the wallet features, not the format used for address generation.

**Comment by FrancisPouliot** [comment]
@luke-jr good point. Safe to assume that all wallets who send from bech32 addresses will also have ability to generate bech32 addresses, and also safe to assume that if they aren't allowing bech32 receving addresses (not even in advanced settings) they also won't allow to send from bech32.
